app/build.gradle

```diff
@@ -69,5 +69,5 @@ android {
 dependencies {
     implementation 'com.google.android.material:material:1.9.0'
     implementation 'androidx.appcompat:appcompat:1.6.1'
-	implementation 'androidx.databinding:viewbinding:7.4.1'
+	//implementation 'androidx.databinding:viewbinding:7.4.1'
```

Comment:
We need this as dependency for code assist. because otherwise it give us wired errors. as the project was not created from viewbinding template.